inc/contact-page.php

```diff
@@ -0,0 +1,71 @@
+<?php
+/**
+ * MEOM contact and settings page.
+ *
+ * @package MEOM Dodo
+ */
+
+namespace MEOM\dodo;
```

Comment:
Why dodo is written with small first letter in namespace?

Reply:
Typo, let's fix!

Reply:
Fixed in cbf5148.